### ISSUE TITLE
#265 breaks folding of json responses

### ISSUE THREAD

**Issue by bohni** [AnWeber/vscode-httpyac#269]
With Release 6.12.0 it is not possible anymore to fold JSON responses in output window

Try

    GET https://httpbin.org/json
    Accept: application/json

with 6.11.6 I get the folding marks on mouse hover:
![image](https://github.com/AnWeber/vscode-httpyac/assets/1252254/03b606c3-f764-4903-9655-3d426db060f0)

with 6.12.0 those marks don't appear.
![image](https://github.com/AnWeber/vscode-httpyac/assets/1252254/14a0a447-bab2-4c16-92e7-df503bbbb99d)

**Comment by AnWeber** [comment]
Every now and then I should read the docs in full. The standard behavior with indentation is deactivated by the custom FoldingRangeProvider. I have now created my own IndentationFoldingRangeProvider, which I use for the indented areas.

**Comment by bohni** [comment]
Hi,

unfortunately, the fix hasn't had the desired effect. Even with v6.12.1 I still can't see the markers.

**Comment by AnWeber** [comment]
@bohni Hello, it seems to work for me. Do you have an example or special settings?
<img width="238" alt="image" src="https://github.com/AnWeber/vscode-httpyac/assets/13434739/c566b38f-ac1c-4f7c-b224-1184711ec292">
<img width="409" alt="image" src="https://github.com/AnWeber/vscode-httpyac/assets/13434739/87b6d166-9432-4221-8852-6697e68c7235">

**Comment by bohni** [comment]
Hi,
sorry for the noise... I tested i ton another setup and there I get those markers also.
Will check my faulty setup and repot back, if I found the reason.
Thanks!

**Comment by AnWeber** [comment]
Do you use response output format `exchange` With this setting I found broken folding

**Comment by bohni** [comment]
Yes, you're right. I have `responseViewContent` set to `exchange`. Without that setting (using default) folding is working.
